src/reynir/Greynir.grammar

```diff
@@ -3957,6 +3985,8 @@ Tala_hk_nf → to_et_hk_nf | to_ft_hk_nf
 NlStak_et_p3/fall/kyn →
     NlStak_p3_et/fall/kyn VeikSögnLhÞt_et/fall/kyn? FsRunaEftirNl?  # fulltrúi bankans í nefndinni
     | NlStak_et_p3/fall/kyn Eða NlStak_p3_et/fall              # Þingnefnd eða hluti Alþingismanna getur lagt fram frumvarp
+    | AnnaðhvortSt Nl/fall Eða Nl/fall/kyn                         # Hér er annaðhvort snjór eða sól  
+    | AnnaðhvortSt Nl/fall/kyn Eða Nl/fall                         # Hér er annaðhvort snjór eða sól  
```

Comment:
This is going to be pretty ambiguous in many cases, but maybe that is unavoidable.

Reply:
Yes, this was the best way I could see